File: main/coreplugins/io.sarl.intellij/src/main/java/io/sarl/intellij/SarlIcons.java
package io.sarl.intellij;

import com.intellij.openapi.util.IconLoader;

import javax.swing.*;

public class SarlIcons {
    private static Icon load(String path) {
        return IconLoader.getIcon(path, SarlIcons.class);
    }

    public static final Icon SARL_PLUGIN = load("/icons/sarlIcon.png"); // 256x256
}

File: main/coreplugins/io.sarl.intellij/src/main/kotlin/io/sarl/intellij/SarlIcons.kt
package io.sarl.intellij

import com.intellij.openapi.util.IconLoader

import javax.swing.*

object SarlIcons {

    val SARL_PLUGIN = load("/icons/sarlIcon.png") // 256x256
    private fun load(path: String): Icon {
        return IconLoader.getIcon(path, SarlIcons::class.java)
    }
}
